### ISSUE TITLE
Installing on fresh machine

### ISSUE THREAD

**Issue by RyanHedges** [uptech/homebrew-oss#4]
```bash
Brew tapping "uptech/homebrew-oss"...
==> Tapping uptech/oss
Cloning into '/opt/homebrew/Library/Taps/uptech/homebrew-oss'...
remote: Enumerating objects: 276, done.
remote: Counting objects: 100% (193/193), done.
remote: Compressing objects: 100% (127/127), done.
remote: Total 276 (delta 133), reused 125 (delta 66), pack-reused 83
Receiving objects: 100% (276/276), 4.77 MiB | 5.06 MiB/s, done.
Resolving deltas: 100% (157/157), done.
Tapped 3 formulae (38 files, 11.2MB).
Installing uptech/oss/git-ps...
==> Cloning https://github.com/uptech/git-ps.git
Cloning into '/Users/hedges/Library/Caches/Homebrew/git-ps--git'...
==> Checking out tag 1.1.1
HEAD is now at 2c77c3f Bump version to v1.1.1
==> Installing git-ps from uptech/oss
==> make build
Last 15 lines from /Users/hedges/Library/Logs/Homebrew/git-ps/01.make:
2021-11-22 22:58:23 +0000

make
build

swift build -c release --disable-sandbox
error: terminated(72): /usr/bin/xcrun --sdk macosx --find xctest output:
    xcrun: error: unable to find utility "xctest", not a developer tool or in PATH
    
make: *** [build] Error 1

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/uptech/homebrew-oss/issues
```

I have xcode installed (`Version 13.1 (13A1030d)`). I gave the dev tools installed.

```
% xcode-select -p
/Library/Developer/CommandLineTools
```

Opening Xcode and going to `Preferences` -> `Locations` allows me to select an option for the installed command line tools on my machine.

<img width="823" alt="Screen Shot 2021-11-22 at 3 17 27 PM" src="https://user-images.githubusercontent.com/4007932/142949278-971937c5-e7f0-48e6-8a67-f21930067745.png">

After making this change, and rerunning the install, apple presented me to allow my terminal to access my `~/Desktop`, which seemed odd, but maybe that's part of a new computer.

In the end it was able to install correctly..

```bash
Installing uptech/oss/git-ps...
==> Cloning https://github.com/uptech/git-ps.git
Updating /Users/hedges/Library/Caches/Homebrew/git-ps--git
==> Checking out tag 1.1.1
HEAD is now at 2c77c3f Bump version to v1.1.1
HEAD is now at 2c77c3f Bump version to v1.1.1
==> Installing git-ps from uptech/oss
==> make build
🍺  /opt/homebrew/Cellar/git-ps/1.1.1: 6 files, 590.1KB, built in 23 seconds

```

Calling this out if anyone else runs into this or if there are changes to code or documentation that may help limit confusion.

**Comment by brandonpittman** [comment]
I'm getting an error like this:

```
make
build

swift build -c release --disable-sandbox
warning: /Users/blp/Library/org.swift.swiftpm/configuration is not accessible or not writable, disabling user-level configuration features.
warning: /Users/blp/Library/org.swift.swiftpm/security is not accessible or not writable, disabling user-level security features.
warning: /Users/blp/Library/Caches/org.swift.swiftpm is not accessible or not writable, disabling user-level cache features.
error: 'git-cl-20230210-20419-civg9i': Invalid manifest
/private/tmp/git-cl-20230210-20419-civg9i/Package.swift:4:8: error: no such module 'PackageDescription'
import PackageDescription
       ^
make: *** [build] Error 1
```

I'm wondering if XCode 14 is missing something that older XCodes had?

**Comment by drewdeponte** [comment]
Yesterday, I had a similar discussion with someone about this in the git-ps slack actually, related to git-cl. The conversation was as follows.

## TLDR

Running `xcode-select -s /Applications/Xcode.app/Contents/Developer` fixed the issue.

## Detailed transcript as follows.

Drew
Normally it would be done through an issue on the git hub repo. But since you are here and we have a thread going ask away in this thread.

Brandon Pittman
Sorry for not making issue.
I tried installing it on a fresh machine just now, and I get this Swift error.

```
make
build

swift build -c release --disable-sandbox
warning: /Users/blp/Library/org.swift.swiftpm/configuration is not accessible or not writable, disabling user-level configuration features.
warning: /Users/blp/Library/org.swift.swiftpm/security is not accessible or not writable, disabling user-level security features.
warning: /Users/blp/Library/Caches/org.swift.swiftpm is not accessible or not writable, disabling user-level cache features.
error: 'git-cl-20230210-20419-civg9i': Invalid manifest
/private/tmp/git-cl-20230210-20419-civg9i/Package.swift:4:8: error: no such module 'PackageDescription'
import PackageDescription
       ^
make: *** [build] Error 1
I get this with Homebrew or when building from source. I'm wondering if XCode 14 lacks something that was present in 13?
```

Drew
Hmm. Let me try uninstalling on my machine and re-installing


Drew
I just uninstalled it and reinstalled it on my machine (macOS 13.1 (22C65) with Xcode 14.2 (14C18)) and everything seemed fine.
```
==> Fetching uptech/oss/git-cl
==> Cloning https://github.com/uptech/git-cl.git
Updating /Users/drewdeponte/Library/Caches/Homebrew/git-cl--git
==> Checking out tag v3.3.2
HEAD is now at 1fc02e9 Bump version to v3.3.2
HEAD is now at 1fc02e9 Bump version to v3.3.2
==> Installing git-cl from uptech/oss
==> make build
🍺  /opt/homebrew/Cellar/git-cl/3.3.2: 6 files, 1.4MB, built in 20 seconds
==> Running `brew cleanup git-cl`...
Disable this behaviour by setting HOMEBREW_NO_INSTALL_CLEANUP.
Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`). 
```

Drew
did you install the Swift toolchain from outside of Xcode or something? Or maybe the command line tools aren't installed?

Brandon Pittman
Seems like the previously installed command line tools were fighting against the XCode instance installed afterwards.
Running `xcode-select -s /Applications/Xcode.app/Contents/Developer` fixed it.

Drew
Interesting

Drew
Glad you got it figured out

**Comment by drewdeponte** [comment]
I think that the Homebrew check to make sure that Xcode is installed doesn't make sure that the command line tools are properly installed or something.
